RadeonRays/src/accelerator/sse2neon.h

```diff
@@ -0,0 +1,6411 @@
+#ifndef SSE2NEON_H
+#define SSE2NEON_H
+
+// This header file provides a simple API translation layer
+// between SSE intrinsics to their corresponding Arm/Aarch64 NEON versions
+//
+// This header file does not yet translate all of the SSE intrinsics.
+//
+// Contributors to this work are:
```

Comment:
Where did this file come from? A link to a repo and its revision would be nice.

Reply:
Added the repo link and revision in the comment.